tools/redfish-gen/redfish/v1/Chassis/ModuleNode.yaml.back

```diff
@@ -0,0 +1,35 @@
+ChassisNode@dynamic:
```

Comment:
This is backup file? Likely it should be removed from git.

Reply:
Yes